lineman/app/app.coffee

```diff
@@ -47,6 +47,7 @@ angular.module('loomioApp').controller 'AppController', ($scope, $filter, $rootS
     {path: '/inbox', component: 'inboxPage' },
     {path: '/groups', component: 'groupsPage' },
     {path: '/profile', component: 'profilePage'},
+    {path: '/email_settings', component: 'emailSettingsPage' },
```

Comment:
Hannah if you change your route to /email_preferences (you can just change the route name and leave the component as is) does that mean we don't need to add a route in here?